android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -231,6 +232,21 @@ public void sendNotification(Bundle bundle) {
                 notification.setVibrate(new long[]{0, vibration});
             }
 
+            int badge = -1;
+            if (bundle.containsKey("badge")) {
+                try {
+                    String badgeAsString = bundle.getString("badge");
```

Comment:
This seems wrong; the payload should contain `badge` as a numeric, not a string.